src/util/cert/libcrypto/cert.c

```diff
@@ -406,16 +416,16 @@ static int sss_rsa_get_key(const EVP_PKEY *cert_pub_key,
 
     RSA_get0_key(rsa_pub_key, tmp_n, tmp_e, NULL);
```

Comment:
Hi,

I'm sorry for all the typos, I was under the assumption the one of the CI systems is still using OpenSSL-1.1 so I assumed that everything was fine if all CI builds succeeded.

In the patch for sssd-2-9 you have `+ RSA_get0_key(rsa_pub_key, &tmp_n, &tmp_e, NULL);` and it looks like I need this for master too, to not get `incompatible-pointer-types` warnings.

bye,
Sumit

Reply:
@mastersin, ping

Reply:
Sorry, it looks like my merge mistake. I fixed it and pushed with force.